File: Backend_Ecommerce/src/main/java/com/commerce/datamodel/UserInfo.java
package com.commerce.datamodel;

import jakarta.persistence.Access;
import jakarta.persistence.AccessType;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.OneToMany;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(schema = "public", name = "userinfo")
@Getter
@Setter
@Access(AccessType.FIELD)
public class UserInfo
{
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Column(name = "id_userinfo")
  private int id;
  
  @Column(name = "address")
  private String address;
  
  @Column(name = "city")
  private String city;
  
  @Column(name = "code_postal")
  private int code_postal;
  
  @Column(name = "country")
  private String country;
  
  @ManyToOne
  @JoinColumn(name = "id_user", referencedColumnName = "id_user")
  private User user;
}

File: C# Backend/E-commerce/E-commerce/Models/UserInfo.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace E_commerce.DataModel
{
    [Table("userinfo", Schema = "public")]
    public class UserInfo
    {
        [Key]
        [Column("id_userinfo")]
        public int Id { get; set; }

        [Column("address")]
        public string Address { get; set; }

        [Column("city")]
        public string City { get; set; }

        [Column("code_postal")]
        public int CodePostal { get; set; }

        [Column("country")]
        public string Country { get; set; }

        [ForeignKey("user")]
        [Column("id_user")]
        public int UserId { get; set; }

    }
}
